src/js/actions/AppsActions.js

```diff
@@ -102,7 +102,11 @@ var AppsActions = {
         });
       });
   },
-  scaleApp: function (appId, instances) {
+  scaleApp: function (appId, instances, force) {
+    var url = `${config.apiURL}v2/apps/${appId}`;
```

Comment:
No need for this variable. You can just initalise `force` with a default value and use it in the url assignment:
```
scaleApp: function (appId, instances, force = false) {
  this.request({
    method: "PUT",
    data: {
      instances: instances
    },
    headers: {
      "Content-Type": "application/json"
    },
    url: `${config.apiURL}v2/apps/${appId}?force=${force}`
  })
  ...
```